### ISSUE TITLE
#786 Missing Samples folder from the RC3.0 dmg. Missing files from models folder in RC3.0 executable,


### ISSUE THREAD

**Issue by nikhilgupta10** [gridlab-d/gridlab-d#786]
The .dmg installation for Mac does not include the Samples folder upon completion. Also several files were added to models folder in the 3.0 repository to be added to the Samples folder. Installation of the windows release candidate revealed that these .glm files were not added to the Samples folder.  

,

**Comment by jcfuller1** [comment]
Please check on this as part of v4.0 release.

**Comment by jcfuller1** [comment]
@afisher1 & @ntenney - we addressed this as part of moving around the example folder, correct?

**Comment by afisher1** [comment]
Yes. I added several more example models to the models folder and removed bad models that existed in there for the 3.2 release. @ntenney can you double check that you are pulling all the files that exist in the models folder into the example forlder in the installer?

**Comment by trevorhardy** [comment]
Closing as a macOS X specific release distributed as a .dmg has not been created for a while.
